src/components/timer/Pie.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+export default class Pie extends React.Component {
```

Comment:
You should be able to implement this as a stateless component since it accepts all mutable state exists in the Timer class.

Reply:
Agreed.